src/test/common/test_mini_flat_map.cc

```diff
@@ -57,6 +80,55 @@ TEST(mini_flat_map, copy_operator_and_element_access) {
   ASSERT_TRUE(m.contains(0));
 }
 
+TEST(mini_flat_map, fmt_formatting) {
+  mini_flat_map<Key, Value> m(4);
+  m[0] = Value(100);
+  m[2] = Value(200);
+  EXPECT_EQ("{0:100,2:200}", m.fmt_print());
+  // compare to operator<<
+  std::ostringstream oss;
+  oss << m;
+  EXPECT_EQ(m.fmt_print(), oss.str());
+  // use indirectly in fmt::format
+  EXPECT_EQ("{0:100,2:200}", fmt::format("{}", m));
+}
+
+TEST(mini_flat_map, fmt_formatting_empty) {
+  mini_flat_map<Key, Value> m(10);
+  EXPECT_EQ("{}", m.fmt_print());
+  // compare to operator<<
+  const auto as_fmt = fmt::format("{}", m);
+  EXPECT_EQ("{}", as_fmt);
+  std::ostringstream oss;
+  oss << m;
+  EXPECT_EQ(as_fmt, oss.str());
+}
+
+TEST(mini_flat_map, fmt_formatting_one) {
+  mini_flat_map<Key, Value> m(4);
+  m[2] = Value(100);
+  const auto using_fmt = fmt::format("{}", m);
+  EXPECT_EQ("{2:100}", using_fmt);
+  // compare to operator<<
+  std::ostringstream oss;
+  oss << m;
+  EXPECT_EQ(using_fmt, oss.str());
+}
+
+TEST(mini_flat_map, fmt_formatting_full) {
+  mini_flat_map<Key, Value> m(4);
+  m[3] = Value(300);
+  m[2] = Value(200);
+  m[1] = Value(100);
```

Comment:
Randomizing the lines could demonstrate the ordering of keys (some people prefer reading unit tests over documentation).